### PR TITLE
Switch to Glossarist gem

### PR DESCRIPTION
**Pull request by skalee**
Concept models are now maintained in [Glossarist](https://github.com/glossarist/glossarist-ruby) gem. This pull request removes old logic from IEV termbase generator and replaces it with calls to Glossarist API.

Fixes glossarist/glossarist-ruby#1.

### CODE REVIEW

**Comment by ronaldtse** [comment]
@skalee thank you! Please merge when ready 👍

File: lib/iev/termbase/term_builder.rb [hunk 1]
```diff
@@ -389,6 +374,21 @@ def extract_entry_status
         end
       end
 
+      def extract_classification
```
[the remark this answers is not in the document]
Comment (skalee) [reply lib/iev/termbase/term_builder.rb:377]: This check is disabled; bug in Hound CI.

File: lib/iev/termbase/term_builder.rb [hunk 2]
```diff
@@ -389,6 +374,21 @@ def extract_entry_status
...
+        classification_val = find_value_for("SYNONYM1STATUS")
+
+        case classification_val
+        when ""
+          "admitted"
+        when "认可的", "допустимый", "admitido"
+          "admitted"
+        when "首选的", "suositettava", "suositeltava", "рекомендуемый", "preferente"
```
[the remark this answers is not in the document]
Comment (skalee) [reply lib/iev/termbase/term_builder.rb:385]: Leaving as it is for now.

**Comment by skalee** [comment]
These test failures are odd. The don't happen locally. Need to investigate.

**Comment by skalee** [comment]
Subtle Glossarist's incompatibility with Ruby 2.6. Reasons:

- https://github.com/glossarist/glossarist-ruby/blob/042aa425e68372ac4f3f003e354638be7c7cfef1/lib/glossarist/concept.rb#L46
- https://bugs.ruby-lang.org/issues/10118

Later I'll create a proper fix for Glossarist gem. For now I'll rather upgrade Ruby in this project.

**Comment by ronaldtse** [comment]
@skalee FYI that @opoudjis will be using this gem in Metanorma in the next 2 weeks. Metanorma is currently bound to Ruby 2.4, so we will have to make the compatibility change.

**Comment by skalee** [comment]
> Metanorma is currently bound to Ruby 2.4, so we will have to make the compatibility change.

Yes, I know this requirement. Compatibility fix is easy, it's just using `Hash#merge` instead of double splat. However, apparently tests are insufficient or incorrect, moreover I may need to add some constraints or coercions to prevent from using incorrect objects as language codes.

**Pull request merged**